libs/ledger-live-common/src/families/aptos/js-buildTransaction.ts

```diff
@@ -21,7 +20,15 @@ const buildTransaction = async (
 
   const txPayload = getPayload(transaction.recipient, amount);
   const txOptions = normalizeTransactionOptions(transaction.options);
-  const tx = await aptosClient.generateTransaction(account.freshAddress, txPayload, txOptions);
+  const tx = await aptosClient.generateTransaction(
+    account.freshAddress,
+    {
```

Comment:
I suggest we need to update `getPayload` function, properties names, rather do double conversions